docs/doc/20-develop/03-java.md

```diff
@@ -0,0 +1,127 @@
+---
+title: How to Work With Databend in Java
+sidebar_label: Java
+description:
+    How to work with Databend in Java.
+---
+
+## Before You Begin
+
+* **Databend :** Make sure Databend is running and accessible, see [How to deploy Databend](/doc/deploy).
+* [How to Create User](../30-reference/30-sql/00-ddl/30-user/01-user-create-user.md)
+* [How to Grant Privileges to User](../30-reference/30-sql/00-ddl/30-user/10-grant-privileges.md)
+
+## Create Databend User
+
+```shell
+mysql -h127.0.0.1 -uroot -P3307
+```
+
+### Create a User
+
+```sql
+CREATE USER user1 IDENTIFIED BY 'abc123';
+```
+
+### Grants Privileges
+
+Grants `ALL` privileges to the user `user1`:
+```sql
+GRANT ALL on *.* TO user1;
+```
+
+## Java
+
+This guideline show how to connect and query to Databend using JDBC. We will be creating a table named `books` and insert a row, then query it.
```

Comment:
This topic shows how to connect and query Databend using JDBC. We will create a table named `books`,  insert a row, and then query data from the table.